sql/pg_duckdb--0.0.1.sql

```diff
@@ -117,6 +117,7 @@ CREATE TABLE secrets (
     region TEXT,
     endpoint TEXT,
     r2_account_id TEXT,
+    use_ssl BOOLEAN,
```

Comment:
nitpick, seems nicer to set a default here:
```suggestion
    use_ssl BOOLEAN DEFAULT true,
```

Reply:
Good idea, Thanks for your suggestion.